Integration test failure due to traceback:

```
Traceback (most recent call last):
  File "/tmp/ansible__ueew9/ansible_module_aws_waf_condition.py", line 656, in <module>
    main()
  File "/tmp/ansible__ueew9/ansible_module_aws_waf_condition.py", line 646, in main
    (changed, results) = condition.ensure_condition_present()
  File "/tmp/ansible__ueew9/ansible_module_aws_waf_condition.py", line 591, in ensure_condition_present
    return self.find_and_update_condition(condition_set_id)
  File "/tmp/ansible__ueew9/ansible_module_aws_waf_condition.py", line 570, in find_and_update_condition
    missing = self.find_missing(update, current_condition)
  File "/tmp/ansible__ueew9/ansible_module_aws_waf_condition.py", line 559, in find_missing
    current_conditions = current_condition[self.conditiontuples]
KeyError: 'IPSetDescriptors'
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -91,6 +91,7 @@ Ansible Changes By Release
   * aws_region_facts
   * aws_s3_cors
   * aws_ssm_parameter_store
+  * aws_waf_condition
   * cloudfront_distribution
   * ec2_ami_facts
   * ec2_asg_lifecycle_hook

--- a/lib/ansible/modules/cloud/amazon/aws_waf_condition.py
+++ b/lib/ansible/modules/cloud/amazon/aws_waf_condition.py
@@ -557,8 +557,8 @@ class Condition(object):
             found = False
             desired_condition = desired[self.conditiontuple]
             current_conditions = current_condition[self.conditiontuples]
-            for current_condition in current_conditions:
-                if not compare_policies(current_condition, desired_condition):
+            for condition in current_conditions:
+                if not compare_policies(condition, desired_condition):
                     found = True
             if not found:
                 missing.append(desired)

